src/bref-local

```diff
@@ -16,19 +16,36 @@ if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
     require_once __DIR__ . '/../../../autoload.php';
 }
 
-$handler = $argv[1] ?? '';
-$data = $argv[2] ?? null;
-$opts = getopt('', ['path:']);
+$opts = getopt('f:', ['path:', 'data:']);
+$handler = $opts['f'] ?? null;
+$data = $opts['data'] ?? null;
+
+// Retains backward compatibilty with `./vendor/bin/bref-local <handler> <event-data>`
+if (empty($opts)) {
+    $handler = $argv[1] ?? '';
+    $data = $argv[2] ?? null;
+}
 
 if (isset($opts['path'])) {
     if (! file_exists($opts['path'])) {
         throw new Exception('The file ' . $opts['path'] . ' could not be found.');
     }
 
-    $handler = $argv[array_key_last($argv)];
     $data = file_get_contents($opts['path']);
 }
```

Comment:
```suggestion
} elseif (isset($opts['data'])) {
    $data = $opts['data'];
}
```

But really I think this whole thing should be refactored using null coalescing instead of `isset(...)`.